libraries/collectd_service.rb

```diff
@@ -104,6 +104,13 @@ def action_enable
             notifies :restart, new_resource, :delayed
           end
 
+          # Installing package starts collectd service automatically
+          # Disable this so that collectd can be managed through poise-service
+          service new_resource.package_name do
+            action [:disable, :stop]
```

Comment:
I also don't think this is going to work all that well. We're going to just end up stopping and disabling our service after the first run. We probably need to ensure that it doesn't start during package installation/upgrade and then remove the sysvinit script.